backup/backup.go

```diff
@@ -188,6 +188,181 @@ func (c *BackupCommand) BackupInfo(cliConnection plugin.CliConnection, backupId
+
+	if resp.Status == "200 OK" {
+		fmt.Println(AddColor("OK", green))
+		//var temp = string(body)
+
+		//var lines = strings.Split(temp, ",")
+		var flag bool
+
+		table.SetHeader([]string{AddColor("backup_guid", white), AddColor("username", white), AddColor("type", white), AddColor("trigger", white), AddColor("started_at", white), AddColor("finished_at", white)})
+
+		var response []interface{}
+		if err := json.Unmarshal(body, &response); err != nil {
+			fmt.Println("Invalid response for the request ", err)
+		}
+
+		var no_of_columns int = 6
+		var field = make([]string, no_of_columns)
+
+		for backup := range response {
+			if strings.Contains(guid, (response[backup].(map[string]interface{}))["instance_guid"].(string)) {
+				field[1] = (response[backup].(map[string]interface{}))["username"].(string)
+				field[2] = (response[backup].(map[string]interface{}))["type"].(string)
+				field[0] = (response[backup].(map[string]interface{}))["backup_guid"].(string)
+				field[0] = AddColor(field[0], cyan)
+				field[3] = (response[backup].(map[string]interface{}))["trigger"].(string)
+				field[4] = (response[backup].(map[string]interface{}))["started_at"].(string)
+				field[5], flag = (response[backup].(map[string]interface{}))["finished_at"].(string)
+				if flag == false {
+					field[5] = "null"
+				}
+				table.Append(field)
+			}
+		}
+
+	}
+	table.Render()
+	if resp.Status != "200 OK" {
+		fmt.Println(AddColor("FAILED", red))
+		var message string = string(body)
+		var parts []string = strings.Split(message, ":")
+		fmt.Println(parts[2])
+	}
+
+}
+
+func (c *BackupCommand) ListDeletedBackupsByInstanceName(cliConnection plugin.CliConnection, serviceInstanceName string) {
+	fmt.Println("Getting the list of  backups in the org", AddColor(helper.GetOrgName(helper.ReadConfigJsonFile()), cyan), "/ space", AddColor(helper.GetSpaceName(helper.ReadConfigJsonFile()), cyan), "/ service instance", AddColor(serviceInstanceName, cyan), "...")
+
+	if helper.GetAccessToken(helper.ReadConfigJsonFile()) == "" {
+		errors.NoAccessTokenError("Access Token")
+	}
+
+	client := GetHttpClient()
+
+	var userSpaceGuid string = helper.GetSpaceGUID(helper.ReadConfigJsonFile())
+
+	var guid string = guidTranslator.FindDeletedInstanceGuid(cliConnection, serviceInstanceName, nil, "")
+	if guid == "Multiple_Instance_Guid" {
+		fmt.Println(AddColor("FAILED", red))
+		fmt.Println("" + serviceInstanceName + " contains multiple GUIDs, please use 'cf instance-events' to get instance guid and then use cf list-backup --guid GUID to get details")
+		fmt.Println("Enter 'cf backup' to check the list of commands and their usage.")
+		os.Exit(1)
+	}
+	var apiEndpoint string = helper.GetApiEndpoint(helper.ReadConfigJsonFile())
+	var broker string = GetBrokerName()
+	var extUrl string = GetExtUrl()
+
+	apiEndpoint = strings.Replace(apiEndpoint, "api", broker, 1)
+
+	req, err := http.NewRequest("GET", apiEndpoint+extUrl+"/backups"+"?space_guid="+userSpaceGuid, nil)
```

Comment:
Again, as stated in [this comment](https://github.com/cloudfoundry-incubator/service-fabrik-cli-plugin/pull/2/files#r197686504), as the instance is deleted, instance_id should be passed, so that broker only returns backups of this instance guid.